src/components/Observations/ObsStatus.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import classnames from "classnames";
+import checkCamelAndSnakeCase from "components/ObsDetails/helpers/checkCamelAndSnakeCase";
+import {
+  ActivityCount,
+  QualityGradeStatus
+} from "components/SharedComponents";
+import { View } from "components/styledComponents";
+import type { Node } from "react";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { useTheme } from "react-native-paper";
+import Observation from "realmModels/Observation";
+
+type Props = {
+  item: typeof Observation,
+  layout?: "horizontal" | "vertical",
+};
+/* eslint-disable react-native/no-inline-styles */
+const ObsStatus = ( { item, layout = "vertical" }: Props ): Node => {
+  const { t } = useTranslation();
+  const theme = useTheme();
+  const qualityGrade = checkCamelAndSnakeCase( item, "qualityGrade" );
+
+  const getIconColor = () => {
+    if ( layout === "horizontal" ) {
+      return theme.colors.onSecondary;
+    }
+    if ( item.viewed === false ) {
+      return theme.colors.error;
+    }
+    return theme.colors.primary;
+  };
```

Comment:
Sorry I didn't catch this earlier, but let's specify color in a prop and not in the component. Again, this component doesn't need to know if it's a list of a grid view, just how it needs to lay itself out.